src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -0,0 +1,328 @@
+                setCaseFileLoading(true);
+                setValue(CASE_FILE, currentFile);
+
+                // Create new case
+                createCaseWithoutDirectoryElementCreation(currentFile)
+                    .then((newCaseUuid) => {
+                        const prevCaseUuid = caseUuid;
+
+                        if (prevCaseUuid && prevCaseUuid !== newCaseUuid) {
+                            deleteCase(prevCaseUuid).catch((error) =>
+                                handleApiCallError(error)
+                            );
+                        }
+                        setValue(CASE_UUID, newCaseUuid, {
+                            shouldValidate: true,
+                        });
+
+                        getCurrentCaseImportParams(newCaseUuid);
+                    })
+                    .catch(handleApiCallError)
+                    .finally(() => {
+                        setCaseFileLoading(false);
+                    });
+            } else {
+                setError(CASE_FILE, {
+                    type: 'caseFileSize',
+                    message: intl
+                        .formatMessage(
+                            {
+                                id: 'uploadFileExceedingLimitSizeErrorMsg',
+                            },
+                            {
+                                maxSize: MAX_FILE_SIZE_IN_MO,
+                            }
+                        )
+                        .toString(),
+                });
+            }
+        }
+    };
+
+    // handle check studyName
+    const [studyNameAdornment, studyNameChecking] = useNameCheck({
+        field: STUDY_NAME,
+        value: studyName,
+        elementType: ElementType.STUDY,
+        setError,
+    });
+
+    /* Effects */
+    // handle create study from existing case
+    useEffect(() => {
+        if (providedExistingCase) {
+            const { elementUuid } = providedExistingCase;
+            setValue(CASE_FILE, providedExistingCase);
+            setValue(CASE_UUID, elementUuid);
+
+            getCurrentCaseImportParams(elementUuid);
+        }
+    }, [getCurrentCaseImportParams, providedExistingCase, setValue]);
+
+    return (
+        <CustomMuiDialog
+            titleId={'createNewStudy'}
+            formSchema={createStudyDialogFormValidationSchema}
+            formMethods={createStudyFormMethods}
+            removeOptional={true}
+            open={open}
+            onClose={onClose}
+            onSave={handleCreateNewStudy}
+            onCancel={handleDeleteCase}
+            disabledSave={!isValid || studyNameChecking}
+        >
+            <Grid container spacing={2} marginTop={'auto'} direction="column">
+                <Grid item>
+                    <StudyNamePrefilledInput
+                        fieldName={STUDY_NAME}
+                        label={'nameProperty'}
+                        adornment={studyNameAdornment}
+                    />
```

Comment:
Do we need this now prefilling is handled by RHF ?
TextInput should do the job

Reply:
after talking with @flomillot , we need to keep this component.